packages/web/src/lib/web-mode.ts

```diff
@@ -74,17 +75,17 @@ export class WebMode {
       path: '/v2/transactions/params'
     });
     const s: SuggestedParams = {
-      fee: txParams.fee,
-      genesisHash: txParams["genesis-hash"],
-      genesisID: txParams["genesis-id"],
-      firstRound: txParams[LAST_ROUND],
+      fee: txParams.fee as number,
+      genesisHash: txParams["genesis-hash"] as string,
+      genesisID: txParams["genesis-id"] as string,
+      firstRound: txParams[LAST_ROUND] as number,
       lastRound: Number(txParams[LAST_ROUND]) + 1000,
       flatFee: false
     };
 
     s.flatFee = userParams.totalFee !== undefined;
-    s.fee = userParams.totalFee ?? userParams.feePerByte ?? txParams["min-fee"];
-    if (s.flatFee) s.fee = Math.max(s.fee, txParams["min-fee"]);
+    s.fee = userParams.totalFee ?? userParams.feePerByte ?? txParams["min-fee"] as number;
```

Comment:
I think we'll probably need to update these, because if we're using `??` in web, then import still won't work in webapp.

Reply:
Thanks for catching